.github/workflows/scan_released.yml

```diff
@@ -0,0 +1,75 @@
+name: Scan released app and container
+on:
+  schedule:
+    - cron: '0 0 * * *' # Run every day at 00:00 UTC.
+env:
+  RELEASED_VERSION: 0.4.1
```

Comment:
I would suggest that we take this directly from github's API so we don't forget to change. The following code should do the trick:

```
curl https://api.github.com/repos/freedomofpress/dangerzone/releases | jq '.[] | select(.draft==false and .prerelease==false) | .tag_name' | head -n 1
"v0.4.1"
```

Maybe also using [env variables like](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) `GITHUB_REPOSITORY_OWNER` and `GITHUB_REPOSITORY` in case someone wants to fork the project and run this automation there as well.

Reply:
> > I would suggest that we take this directly from github's API so we don't forget to change.
> 
> Nice! I wanted to leave this as a todo item, but you found an elegant solution. I can incorporate it, thanks.

There is an even more elegant one, apparently:


```bash
curl https://api.github.com/repos/freedomofpress/dangerzone/releases/latest | jq '.tag_name'
```

>>In case someone wants to fork the project and run and see this automation output we could use generic [env variables like](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) GITHUB_REPOSITORY_OWNER and GITHUB_REPOSITORY instead of hardcoding ours?
>
>I think this may end up breaking this action for other users, since by forking our repo, they don't also copy our releases. So, they won't be able to get the latest releast tag, nor the tarball. If they want to somehow mirror those in their fork, I guess they would change this file as well. On the other hand, if they simply want to run this action, they don't need to touch anything. Does this make sense?

You're right! Then let's leave that hardcoded then.